internal/pkg/project/manifest/file.go

```diff
@@ -69,7 +69,7 @@ func loadFile(fs filesystem.Fs) (*file, error) {
 
 	// Validate
 	if err := content.validate(); err != nil {
-		return nil, err
+		return content, err
```

Comment:
V pripade chyby, vratime obsah manifestu, ... mozeme z neho zobrat to co je validne, napr. nastavenie branches. VId dalej v testoch.